internal/vlan/retrieve.go

```diff
@@ -41,19 +41,28 @@ func (c *Client) Retrieve() *cobra.Command {
 
 		RunE: func(cmd *cobra.Command, args []string) error {
 			cmd.SilenceUsage = true
-			vnets, _, err := c.Service.List(projectID, c.Servicer.ListOptions(nil, nil))
+			request := c.Service.FindVirtualNetworks(cmd.Context(), projectID).Include(nil).Exclude(nil)
```

Comment:
In this case, `Include` and `Exclude` need to be wired up to the command-line flags to match the old behavior

Reply:
This was marked resolved but `Include` and `Exclude` are still not wired up to the command-line flags.

Reply:
Looks like they're in there now.